posthog/api/user.py

```diff
@@ -213,7 +340,10 @@ def redirect_to_site(request):
 @require_http_methods(["PATCH"])
 @authenticate_secondarily
 def change_password(request):
-    """Change the password of a regular User."""
+    """
+    DEPRECATED: This endpoint has been deprecated in favor of /api/v2/user/ 
+    and will be removed in PostHog V2.
```

Comment:
I wouldn't tie this to PostHog V2, since it may never happen 🤔  or may happen soon for an HN relaunch or something